test/benchmarks.cpp

```diff
@@ -70,7 +68,12 @@ void benchmarkLoopVariable(std::function<void(QInterfacePtr, int)> fn, bitLenInt
 
     double avgt, stdet;
 
-    for (numBits = 4; numBits <= mxQbts; numBits++) {
+    bitLenInt mnQbts = 4;
+    if (single_qubit_run) {
+        mnQbts = mxQbts;
+    }
+
```

Comment:
Use an else instead of the default value being overridden

Reply:
Also, choose snake or camel but not both

Reply:
The "else" makes sense. The way we originally wrote this, snake is exclusively `extern` variables and camel is everything else. That's followed, here; can it stand?

Reply:
:+1: I forgot we had that consistent scheme.  As long as it’s consistent, it’s good :)